en/firefox/home.ftl

```diff
@@ -0,0 +1,47 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+### URL: https://www-dev.allizom.org/firefox/
+
+firefox-home-firefox-protect-your = { -brand-name-firefox } - Protect your life online with privacy-first products
+firefox-home-firefox-is-more-than = { -brand-name-firefox } is more than a browser. Learn more about { -brand-name-firefox } products that handle your data with respect and are built for privacy anywhere you go online.
+firefox-home-the-browser-is-just = The browser is just the beginning
+firefox-home-meet-our-family-of = Meet our family of products
+firefox-home-get-trackers-off = Get 2,000+ trackers off your trail — including { -brand-name-facebook }
+firefox-home-know-when-hackers-strike = Know when hackers strike — and stay a step ahead
+firefox-home-start-getting-breach = Start getting breach reports
+firefox-home-keep-your-passwords = Keep your passwords safe on every device
+firefox-home-learn-more-about-lockwise = Learn more about { -brand-name-lockwise }
+
+# The strong tags around "respect" add a special underline. The underline breaks if it is on two words, so please omit the strong tags if they need to be around multiple words in your language.
+firefox-home-get-the-respect-you = Get the <strong>respect</strong> you deserve
+
+firefox-home-every-single-firefox = Every single { -brand-name-firefox } product honors our Personal Data Promise: <strong>Take less. Keep it safe. No secrets.</strong>
+firefox-home-share-large-files-without = Share large files without prying eyes
+firefox-home-start-sending-files = Start sending files safely
+firefox-home-trade-clickbait-for = Trade clickbait for quality content
+firefox-home-learn-more-about-pocket = Learn more about { -brand-name-pocket }
+
+# The strong tags around "privacy" add a special underline. The underline breaks if it is on two words, so please omit the strong tags if they need to be around multiple words in your language.
+firefox-home-one-login-all-your = One login. All your devices. A family of products that respect your <strong>privacy</strong>.
+
+firefox-home-join-firefox = Join { -brand-name-firefox }
```

Comment:
@peiying2 please note there's one new string here which is currently [untranslated in production](https://www.mozilla.org/de/firefox/). I believe this bug is due to the page relying on a .lang string that was removed from the L10n-repo pre-swtichover.

I've added it back here, so locales can get it translated.